sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -12346,7 +12346,7 @@
           },
           "properties": {},
           "calculateServer": true,
-          "calculateValue": "/**\r\n * Full-time student will only be considered independant if\r\n * parent information does not need to be entered (showParentInformation) or \r\n * if parent information does not need to be entered and student selects\r\n * both the parents passed away \r\n * or estranged from both of my parents. \r\n * Otherwise, student is considered dependant.\r\n */\r\nif (data.howWillYouBeAttendingTheProgram === \"Part Time\") {\r\n\tinstance.setValue(\"independant\");\r\n} else if (data.showParentInformation) {\r\n\tconst doesNotHaveParents = (data.parentbcResident === \"bothOfMyHaveParentsHavePassedAwayAndIHaveNoLegalGuardian\" || data.parentbcResident === \"estranged\");\r\n\tinstance.setValue(doesNotHaveParents ? \"independant\" : \"dependant\");\r\n} else {\r\n\tinstance.setValue(\"independant\");\r\n}\r\n",
+          "calculateValue": "/**\r\n * Full-time student will only be considered independant if\r\n * parent information does not need to be entered (showParentInformation) or \r\n * if parent information does not need to be entered and student selects\r\n * both the parents passed away \r\n * or estranged from both of my parents. \r\n * Otherwise, student is considered dependant.\r\n */\r\nif (data.howWillYouBeAttendingTheProgram === \"Part Time\") {\r\n\tinstance.setValue(\"independant\");\r\n} else if (data.showParentInformation) {\r\n\tconst doesNotHaveParents = (data.parentbcResident === \"bothOfMyHaveParentsHavePassedAwayAndIHaveNoLegalGuardian\" || data.parentbcResident === \"estranged\");\r\n\tinstance.setValue(doesNotHaveParents ? \"independant\" : \"dependant\");\r\n} else {\r\n\tinstance.setValue(\"dependant\");\r\n}\r\n",
```

Comment:
Let me know if I am missing something. if the `showParentInformation` is false. Then the student is considered as independent but with this logic they will be considered as dependant. 

![image](https://github.com/user-attachments/assets/2ce17c49-7a66-4d3f-931b-a1c03435cddb)